test/integration/infrastructure/infrastructure_test.go

```diff
@@ -313,6 +316,16 @@ var _ = Describe("Infrastructure tests", func() {
 			}
 			Expect(c.Create(ctx, cluster)).To(Succeed())
 
+			priorityClass = &schedulingv1.PriorityClass{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: v1beta1constants.PriorityClassNameShootControlPlane300,
+				},
+				Description:   "PriorityClass for Shoot control plane components",
+				GlobalDefault: false,
+				Value:         999998300,
+			}
+			Expect(client.IgnoreAlreadyExists(c.Create(ctx, priorityClass))).To(BeNil())
+
```

Comment:
Shouldn't this be inside the `runTest` function here,
https://github.com/gardener/gardener-extension-provider-aws/blob/5e67cfc2354fdd1017479452589893894f7b4201/test/integration/infrastructure/infrastructure_test.go#L367
I see lot of code is duplicated here which is in runTest, can we reuse them?

Reply:
Can we not simply put this logic into `BeforeSuite`? It's a global/non namespaced resource, which is needed by multiple test cases.

Reply:
Yes. Better.